truffle-seq/OwnerManagedLite_test_add_owner.md

```diff
@@ -0,0 +1,1329 @@
+      header {
+        HorizontalAlignment left
+        FontColor purple
+        FontSize 14
+        Padding 10
+      }
+    </style>
+
+header Insights by Truffle
+
+title Txn Hash: 0x60540e93316225462f2be8151e07cddf1be99e766799e3023ecae766dd6192be
+
+
+actor EOA as "EOA"
+participant OwnerManagedLite_01 as "OwnerManagedLite_01"
+
+"EOA" -> "OwnerManagedLite_01" ++: AddState(\n\
+<#FEFECE,#FEFECE>|= type |= name |= value |\n\
+| address | ownerAddress | 0xc633ba530a1D46B6B9Fb88DD1DB07266e43648a8 |\n\
+| bool | isActive | true |\n\
+) { 0.0 GWEI }
+"OwnerManagedLite_01" -> "EOA" --: 
+
+legend
+Participant details
+<#FEFECE,#D0D000>|= Alias |= Contract name |= Address |
+<#FEFECE>| EOA | Externally Owned Account | 0xc633ba530a1D46B6B9Fb88DD1DB07266e43648a8 |
+<#FEFECE>| OwnerManagedLite_01 | OwnerManagedLite | 0x7d125EE2EFBE5f2bEC679f793daFCa20a58194B9 |
+endlegend
+
+@enduml
+```
+
+##### d3, tx: 0x55e167a1cf9aa49b6ec10e76fde2a5beaa456ce293ebc487c475df264979975b
+
+[SVG :telescope:](https://www.planttext.com/api/plantuml/svg/bLHjJzim4FxkNt43QMnbR4wQbmPr5Nr91z8c4A3jmvYcIt8f4QbJEGwqqFttNTAiObhvi2ZosUVpuxlddaI8azAWDjKy5m8hKwXg7f4MvLsc5gXn3ZdDI2KZZEzckgXKCYxoGiF-D9o6ur28VcbMEGq4DCyjOK8Qdjebm6cXiyT26So7UJPJSrA64LEp3PZovWPoKUb5JdzjN6MF19QpzLrWacHg1fPiNRyQs__G9Y7Q-yzKcSrkJGdH2gvrbQQCAqncSeBhfO9JB6-FGIvTbopFHojE0qGdY3oABKc-boPaeniHErql9ZleKXGxFJzsV3T9RSy9_23mtKWmPR7XrCFp8M09Uspt1DDciZXR89Tw_g18Vq65Cqe-PuP-Igi9tE7V4w8v3qU3tVjmU7WCJC2LGKDDm9iRTIFwRIlUjNQm_WXcjI1Wgt3Us7lCAvxKuUiQGbDPmXgA6cIuNSfbx7Mx4RfTYTR4yKRUA9X6lTvaOar6qhSzZvokv_Imju5w2qyWtqluz2qyuqxiJdfvz7rVK_AZhcfJtj4nz5y_e5RSoG7quusUE6RGwLo6NyFBwrUTJlz3un_02pL2QNIjX3NmU_2dXgm4L9ko3jeSHKBSe-oUENk9zet2nSMplYLaCClBPnTFv4HAMPFBGkOMycJC2jKClgLvI-Ns72TQQuF7fI6jCCzNZIOI6COnVqtcFsb_3hnBM_zwcmlynBBTCBJ3wIXqKpiAnvuVf7xGJN0wHbkYsxC2PnJm1Sn4ouSuuP7_2xy1)
+
+
+```plantuml
+
+
+@startuml
+
+autonumber
+skinparam legendBackgroundColor #FEFECE
+
+<style>
+      header {
+        HorizontalAlignment left
+        FontColor purple
+        FontSize 14
+        Padding 10
+      }
+    </style>
+
+header Insights by Truffle
+
+title Txn Hash: 0x55e167a1cf9aa49b6ec10e76fde2a5beaa456ce293ebc487c475df264979975b
+
+
+actor EOA as "EOA"
+participant OwnerManagedLite_01 as "OwnerManagedLite_01"
+
+"EOA" -> "OwnerManagedLite_01" ++: addStateOwner(\n\
+<#FEFECE,#FEFECE>|= type |= name |= value |\n\
+| address | ownerAddress | 0xc633ba530a1D46B6B9Fb88DD1DB07266e43648a8 |\n\
+) { 0.0 GWEI }
+"OwnerManagedLite_01" x-[#red]-> "EOA" --: <&warning> <color #red>**REVERT!**</color> (\n\
+<#FEFECE,#FEFECE>| string |  | 'Address is an owner' |\n\
+)
+deactivate "OwnerManagedLite_01"
+
+legend
+Participant details
+<#FEFECE,#D0D000>|= Alias |= Contract name |= Address |
+<#FEFECE>| EOA | Externally Owned Account | 0xc633ba530a1D46B6B9Fb88DD1DB07266e43648a8 |
+<#FEFECE>| OwnerManagedLite_01 | OwnerManagedLite | 0x7d125EE2EFBE5f2bEC679f793daFCa20a58194B9 |
+endlegend
+
+@enduml
+```
+
+
+
+## Adds the owner and is retrievable
```

Comment:
This works as expected 👍  - 

my only concern is that it doesn't show the query afterward to validate the data, which makes it look like a previous test